modules/nf-core/sigprofiler/extractor/main.nf

```diff
@@ -0,0 +1,56 @@
+process SIGPROFILER_EXTRACTOR {
+    tag "$meta.id"
+    label 'process_single'
+
+    // TODO nf-core: List required Conda package(s). Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    conda "${moduleDir}/environment.yml"
+    container "sigprofilerextractor"
+
+    input:
+    tuple val(meta), path(vcf)
+    val(genome)
+
+    output:
+    tuple val(meta), path("results"), emit: results
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def outdir = "results"
+    // TODO nf-core: It MUST be possible to pass additional parameters to the tool as a command-line string via the "task.ext.args" directive
+    """
+    #!/usr/bin/env python
```

Comment:
It is unusual to include a Python script this way, although it is not necessarily wrong. You can consider packaging the script in the container and calling it with command line parameters. Feel free to argue here.